presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -569,6 +569,14 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
         opOut.runtimeStats.emplace(
             statName, createProtocolRuntimeMetric(statName, op.outputVectors));
       }
+      if (op.memoryStats.numMemoryAllocations > 0) {
+        auto statName = fmt::format(
```

Comment:
Can we provide an anonymous utility for this? Like opRuntimeStatsName()? It has been used for more than one places? Thanks!